src/Controls/samples/Controls.Sample/Maui.Controls.Sample.csproj

```diff
@@ -39,6 +39,10 @@
     <DefineConstants>$(DefineConstants);WINDOWS</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
+    <PackageReference Include="Microsoft.Web.WebView2" />
```

Comment:
Why is this needed in the sample?

Reply:
Sample uses WebView2

Reply:
> Why is this needed in the sample? I see it is in every project and I am scared...

yes, shouldn't be necessary with WinAppSdk 1.6 now containing a WebView2 nupkg reference.  investigating...

Reply:
circling back - the WebView2 nupkg transitive reference issue will be fixed in the next release of same (later in September)